bin/pqcheck

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bin/ruby
```

Comment:
~~~
env: bin/ruby: No such file or directory
~~~

I think you mean:
~~~ diff
-#!/usr/bin/env bin/ruby
+#!/usr/bin/env ruby
~~~

Reply:
This assumes $PWD is the root of logstash and that the script is invoked `bin/pqcheck`, so to run this in production someone would have to:

```
cd /usr/share/logstash
bin/pqcheck
```

And running `/usr/share/logstash/bin/pqcheck` would fail with a confusing error message as @yaauie showed

Reply:
I agree but I don't know how to solve this simply. Unless we have practical solutions I think we can live with that caveat since with the on-liner we also had to give instruction WRT the logstash home.

Reply:
this is basically to be better than asking users to copy&paste a one-liner script around.

Reply:
so yeah, the only way IMO would be to split that into 2 files, a bash `pqcheck` and a `pqcheck.rb` with the similar bootstrapping logic as in `bin/ruby` or `bin/logstash-plugins`